packages/core/src/components/Sender/index.vue

```
@@ -242,48 +250,54 @@ function submit() {
+    case 'enter':
+      _isComKeyDown = e.shiftKey || e.metaKey || e.ctrlKey || e.altKey; // 只处理 Enter 键
+      break;
+    default:
+      _isComKeyDown = false; // 默认不处理组合键
+      break;
+  }
+  if (e.keyCode === 13) {
+    e.preventDefault();
+    if (props.submitType === 'enter') {
+      _isComKeyDown ? _resetSelectionRange() : submit();
+    } else {
```

ESLint

[error] Closing curly brace appears on the same line as the subsequent block.

(style/brace-style)